experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -383,8 +383,91 @@ describe("editable-tree: editing", () => {
 		trees[0].context.free();
 	});
 
+	describe(`can move nodes`, () => {
+		it("to the left within the same field", () => {
+			const [provider, trees] = createSharedTrees(
+				getTestSchema(FieldKinds.sequence),
+				[{ type: rootSchemaName }],
+				2,
+			);
```

Comment:
There is no need to create 2 trees here: this isn't a collaboration test, its a test that the API on editableTree does the right thing (not that it also works correctly through collab: there are other tests for that), so we can simplify this test significantly.  At the very least we should have this create one tree instead of 2.

We can do better than that though: See the todo at the top of this file (like 78-80): we can apply that to these new tests. See the pattern in src/test/shared-tree/schematizeTree.spec.ts for an example: You don't need to do the change in this PR, but you can if you want.